apps/react-native-webview-example/app/room/index.tsx

```diff
@@ -21,12 +20,15 @@ export default function Room() {
                     <Text key={index}>{entry}</Text>
                 ))}
             </ScrollView>
+            {/* <Button onPress={() => wherebyRoomRef.current?.knock()} title="Knock" /> */}
             <View style={{ flex: 1, height: "100%" }}>
                 <WherebyEmbed
                     ref={wherebyRoomRef}
                     style={styles.container}
-                    roomUrl={ROOM_URL}
-                    queryParams={queryParams}
+                    room={ROOM_URL}
+                    minimal={"on"}
```

Comment:
Maybe add a comment why the minimal configuration is suggested on mobile, or just as en example of how to use properties.